### PR TITLE
PAULINE MORSLI WEBSEV 0120 [ongoing]

### PR DESCRIPTION
**Pull request by PaulineLalou**

### CODE REVIEW

File: starter_code/queries.md [hunk 1]
```diff
@@ -4,80 +4,72 @@
...
-
-<!-- Your Code Goes Here -->
+filter : { $and: [ {"ipo.valuation_amount": { $gte: "100000000" } }, { founded_year: { $lt: "2010" } } ] }
+project : { name: 1, ipo: 1}
 
 ### 5. All the companies that have less than 1000 employees and have been founded before 2005. Order them by the number of employees and limit the search to 10 companies.
-
-<!-- Your Code Goes Here -->
+filter : { $and: [ { number_of_employees: { $lt: 1000 } }, { founded_year: {$lt: 2005} } ] }
+sort : { number_of_employees: -1}
```
Comment (ta-web-paris) [inline starter_code/queries.md]: You forgot the limit !

File: starter_code/queries.md [hunk 2]
```diff
@@ -4,80 +4,72 @@
...
 
 ### 6. All the companies that don't include the `partners` field.
-
-<!-- Your Code Goes Here -->
+filter : {partners: {$exists: false}}
 
 ### 7. All the companies that have a null type of value on the `category_code` field.
 
-<!-- Your Code Goes Here -->
+filter : { category_code: { $in: [ null ] } }
```
Comment (ta-web-paris) [inline starter_code/queries.md]: It's working but you could make it simpler  this way 
{ category_code: null}
or 
{ category_code: {$eq: null}}

File: starter_code/queries.md [hunk 3]
```diff
@@ -4,80 +4,72 @@
...
 ### 8. All the companies that have at least 100 employees but less than 1000. Retrieve only the `name` and `number of employees` fields.
-
-<!-- Your Code Goes Here -->
+filter : {$and: [ { number_of_employees: {$gte: 100} }, { number_of_employees: {$lt: 1000} } ] }
+project : { name: 1, number_of_employees: 1}
 
 ### 9. Order all the companies by their IPO price in a descending order.
-
-<!-- Your Code Goes Here -->
+sort : { "ipo.valuation_amount": 1}
```
Comment (ta-web-paris) [inline starter_code/queries.md]: Should be { "ipo.valuation_amount": -1}   to sort it in a descending order.

File: starter_code/queries.md [hunk 4]
```diff
@@ -4,80 +4,72 @@
...
 ### 11. All the companies founded on the second semester of the year. Limit your search to 1000 companies.
 
-<!-- Your Code Goes Here -->
-
-<!-- ### 12. All the companies that have been 'deadpooled' after the third year. -->
-
-<!-- Your Code Goes Here -->
-
+filter : {$and: [ { founded_month: {$gte: 7} }, { founded_month: {$lte: 12} } ] }
+limit : 1000
```
Comment (ta-web-paris) [inline starter_code/queries.md]: I don't think there's a 13e mois here.
So you could simplify it as follows:
 { founded_month: {$gte: 7} }

File: starter_code/queries.md [hunk 5]
```diff
@@ -4,80 +4,72 @@
...
-
-<!-- Your Code Goes Here -->
+project : {name: 1, founded_year: 1}
+sort : {founded_year: -1}
 
 ### 15. All the companies that have been founded on the first seven days of the month, including the seventh. Sort them by their `acquisition price` in a descending order. Limit the search to 10 documents.
-
-<!-- Your Code Goes Here -->
+filter : {$and: [ { founded_day: {$gte: 1} }, { founded_day: {$lte: 7} } ] }
+sort : {"acquisition.price_amount": -1}
```
Comment (ta-web-paris) [inline starter_code/queries.md]: I don't think there's a day 0 or below.
You could simplify it by 
{ founded_day: {$lte: 7} }

File: starter_code/queries.md [hunk 6]
```diff
@@ -4,80 +4,72 @@
...
 ### 17. All the companies whose acquisition amount is more than 10.000.000, and currency is 'EUR'.
-
-<!-- Your Code Goes Here -->
+{$and: [ { "acquisition.price_amount": {$gt: 10000000} }, { "acquisitions.price_currency_code": "EUR"} ] }
 
 ### 18. All the companies that have been acquired on the first trimester of the year. Limit the search to 10 companies, and retrieve only their `name` and `acquisition` fields.
-
-<!-- Your Code Goes Here -->
-
+filter : {$and: [ { "acquisition.acquired_month": {$gte: 1} }, { "acquisition.acquired_month": {$lte: 3} } } ]
```
Comment (ta-web-paris) [inline starter_code/queries.md]: I don't think there's a month 0.
 { "acquisition.acquired_month": {$lte: 3} } }

**Comment by ta-web-paris** [comment]
Well done   👌

**Comment by PaulineLalou** [comment]
thanks !

**Pull request closed without merging**